src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -55,24 +55,55 @@ def get_cookie_value(self, name):
 
     @keyword
     def get_cookie(self, name):
-        """Returns the cookie found with `name` in extended variable format.
-
-        If no cookie is found with `name`, this keyword fails.
+        """Returns a cookie object found with ``name``.
+
+        If no cookie is found with ``name``, keyword fails. The cookie object
+        contains details about the cookie. Attributes available in the object
+        are documented in the table below.
+        | = Attribute = |             = Explanation =                            |
+        | domain        | Specifies hosts to which the cookie will be sent       |
+        | expiry        | The maximum lifetime of the cookie as EPOCH            |
```

Comment:
This is a bit confusing. Is this a timeout like 1000 seconds or is it the actual time when the cookie expires? If it's a timeout, reference to EPOCH is wrong. If it's a time, then we should expose it to users as a datetime object.

Also, sentences should end with `.`.

Reply:
It's time when cookie expires and not a timeout. Epoch is returned directly by the Selenium and therefore we just blindly return that and returning something more human readable is good idea. When you are referring to the datetime object are you actually referring to the Python DateTime object? Or should we decide to return some string which is in more human readable format?

Reply:
I'd return Python's [datetime](https://docs.python.org/2/library/datetime.html#datetime.datetime). It has several benefits:

1. Trivial to construct from an EPOCH time with `datetime.fromtimestamp` factory method.
2. Has useful attributes that can be accessed like `${cookie.expiry.day}`.
3. Is supported as an input format by Robot's DateTime library.

Reply:
Done